plane/source/docs/precision-autolanding.rst

```diff
@@ -0,0 +1,139 @@
+
+Precision Altitude Source
+=========================
+
+A normal GPS will only provide vertical accuracy to within a couple of meters. However, if an RTK corrected GPS is used, tens of centimeter accuracy is possible. However, this requires a base station or laptop providing correction data. See :ref:`common-rtk-correction` for details.
+
+To use the GPS as the altitude source instead of BARO set:
+
+:ref:`EK3_SRC1_POSZ<EK3_SRC1_POSZ>` = 3 to use GPS instead of barometer for altitude
+
+Or a downward facing Lidar/Rangefinder can be used. See :ref:`rangefinder-autolanding` page.
+
+Approach Airspeed
+=================
+
+Automatic landing is greatly assisted by the use of an airspeed sensor.
+When using an airspeed sensor the landing approach speed (the speed
+coming down the glide slope) is controlled by the
+:ref:`TECS_LAND_ARSPD <TECS_LAND_ARSPD>`
+parameter, in meters/second.
+
+You need to choose a value for :ref:`TECS_LAND_ARSPD <TECS_LAND_ARSPD>` that is above the
+stall speed of your aircraft, but low enough that the aircraft is able
+to lose altitude and land in a reasonable distance. Note that as the
+stall speed is dependent on the weight of your aircraft you will need to
+adjust the landing speed if you change the aircraft's weight
+significantly (such as by adding batteries or a camera).
+
+To further improve landing you can use a Pre-Flare to reduce airspeed
+just before the flare. This is enabled by setting either
+:ref:`LAND_PF_ALT <LAND_PF_ALT>` or :ref:`LAND_PF_SEC <LAND_PF_SEC>`
+to either enter a pre-flare state at a fixed altitude or at an estimated
+seconds to ground (given your current decent rate). Once the Pre-Flare
+is triggered the desired airspeed becomes :ref:`LAND_PF_ARSPD <LAND_PF_ARSPD>`.
+This value should be lower than :ref:`TECS_LAND_ARSPD <TECS_LAND_ARSPD>` but greater than the
+stall speed. This is particularly useful where reverse thrust is
+available. However, some aircraft can handle a stall landing so setting
+this to a very low number (1) will tell the aircraft to bleed off as
+much airspeed as possible before the flare.
+
+Controlling the approach
+------------------------
+
+During the landing approach the autopilot needs to balance the requested
+airspeed (set by :ref:`TECS_LAND_ARSPD <TECS_LAND_ARSPD>`) and the requested glide slope and
+landing position (set by the previous waypoint and final landing point).
```

Comment:
I think this should be "set by the final approach waypoint" - to use the same terminology we settled on for the main page.